### ISSUE TITLE
Table cell text with break avoid gets cut off 

### ISSUE THREAD

**Issue by PhilNorfleet** [NigelCunningham/pagedjs#1]
I am testing out your forthcoming changes to pagedjs because I am very interested in the table pagination behavior for a project I have in development. I am hoping this set of changes can address my issue:

![Screen Shot 2024-02-02 at 1 39 06 AM](https://github.com/NigelCunningham/pagedjs/assets/8272837/001d23e8-fad2-4a9d-9d19-eaab8c9e4e82)
![Screen Shot 2024-02-02 at 1 39 22 AM](https://github.com/NigelCunningham/pagedjs/assets/8272837/5c5525e8-a951-45e8-b0f5-261c55433918)

As you can see in these screenshots, the final row on the first page has cut the first cell content and continued it on the next page. I want to avoid doing that in the case where this text element would be cut off, so I have applied a page-break-inside: avoid style to that text element but it hasn't had the desired effect. 

I appreciate that this is a work in progress fork for the main repo; I am very interested in your progress and want to understand this package well enough to contribute to resolving this issue.

**Comment by NigelCunningham** [comment]
Would you be able to send me the HTML so I can debug and test locally, please?

**Comment by PhilNorfleet** [comment]
Thank you for the quick reply. It would be tricky to share, as I am building using React/Nextjs and fetching data that requires authentication to fill in the content. I will work on deploying a demo that would let you load the site with some hard coded data to fill in the report to replicate my issue in a browser. If you need access to the app source code itself to run it locally, I can work on sharing that with you via email or some other private channel.

**Comment by NigelCunningham** [comment]
Perhaps you could set a breakpoint in the pagedJS code and grab the DOM that it's trying to render from there? If you search for the layout function, set a breakpoint on the first line and then in the developer console run the following when it's hit, you should be able to copy/paste the result and I can use that:

```
new XMLSerializer().serializeToString(content)
```

(Credit: https://stackoverflow.com/questions/4896944/converting-range-or-documentfragment-to-string)

**Comment by PhilNorfleet** [comment]
Good idea. I've attached the output as the content.txt file and also a copy of the stylesheet I am passing in to the Previewer. Sharing as txt files because of github limitations.
[content.txt](https://github.com/NigelCunningham/pagedjs/files/14144089/content.txt)
[styles.txt](https://github.com/NigelCunningham/pagedjs/files/14144091/styles.txt)

I learned that I was not actually using your PR to generate the above screenshots; I had attempted to use `pnpm patch` to apply the changes from this fork but that turned out not to have worked properly. The above images were generated with the main package, however I was able to get this fork working in my project and encounter a different set of issues.

If you are able to use the attached files to render the report using both the main package pagedjs and this fork, perhaps you can see the differences; in this fork it seems some cells have been removed between pages
